tests/components/androidtv/patchers.py

```diff
@@ -65,7 +65,6 @@ def __init__(self, host=ADB_SERVER_HOST, port=DEFAULT_ADB_SERVER_PORT) -> None:
     async def device(self, serial):
         """Mock the `ClientAsync.device` method when the device is not connected via ADB."""
         self._devices = []
-        return None
```

Comment:
This mock method should match the patched interface:
https://github.com/JeffLIrion/python-androidtv/blob/7aa7c6a84e77a4c6a8941abedd5ac60f949b57e9/androidtv/adb_manager/adb_manager_async.py#L135-L140

Returning None is correct.

Reply:
Yeah, the missing typing was the problem, I fixed it here.